controllers/operands/kubevirt.go

```diff
@@ -680,12 +681,24 @@ func getKVSeccompConfig() *kubevirtcorev1.SeccompConfiguration {
 	}
 }
 
+func getKubevirtCRAnnotations(hc *hcov1beta1.HyperConverged) map[string]string {
+	var annotations map[string]string
+
+	if hc.Spec.FeatureGates.AlignCPUs != nil && *hc.Spec.FeatureGates.AlignCPUs {
+		annotations = make(map[string]string)
+		annotations["alpha.kubevirt.io/EmulatorThreadCompleteToEvenParity"] = ""
+	}
+
+	return annotations
+}
+
 func NewKubeVirtWithNameOnly(hc *hcov1beta1.HyperConverged, opts ...string) *kubevirtcorev1.KubeVirt {
 	return &kubevirtcorev1.KubeVirt{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "kubevirt-" + hc.Name,
-			Labels:    getLabels(hc, hcoutil.AppComponentCompute),
-			Namespace: getNamespace(hc.Namespace, opts),
+			Name:        "kubevirt-" + hc.Name,
+			Labels:      getLabels(hc, hcoutil.AppComponentCompute),
+			Annotations: getKubevirtCRAnnotations(hc),
```

Comment:
Please move to NewKubeVirt function, keeping this one producing the minimal version of KV CR.

Reply:
I agree. moved this logic outside of this function.